test/week_test.js

```diff
@@ -57,6 +57,57 @@ describe('Week', () => {
     assert(day.prop('day').isSame(dayClicked, 'day'))
   })
 
+  it('should call the provided onWeekSelect function and pass the first day of the week', () => {
+    let firstDayReceived = null
+
+    function onWeekClick (newFirstWeekDay) {
+      firstDayReceived = newFirstWeekDay.format()
+    }
+
+    const weekStart = moment('2015-12-20')
+    const week = shallow(
+      <Week day={weekStart} showWeekNumber onWeekSelect={onWeekClick} />
+    )
+    const weekNumberElement = week.find(WeekNumber)
+    weekNumberElement.simulate('click')
+    expect(firstDayReceived).to.equal(weekStart.format())
+  })
+
+  it('should call the provided onWeekSelect function and pass the week number', () => {
+    let weekNumberReceived = null
+
+    function onWeekClick (unused, newWeekNumber) {
+      weekNumberReceived = newWeekNumber
+    }
+
+    const weekStart = moment('2015-12-20')
+    const realWeekNumber = parseInt(weekStart.format('w'), 10)
+    const week = shallow(
+      <Week day={weekStart} showWeekNumber onWeekSelect={onWeekClick} />
+    )
+    const weekNumberElement = week.find(WeekNumber)
+    weekNumberElement.simulate('click')
+    expect(weekNumberReceived).to.equal(realWeekNumber)
+  })
+
+  it('should set the week number with the provided formatWeekNumber function', () => {
+    let firstDayReceived = null
+
+    function weekNumberFormatter (day) {
+      firstDayReceived = day.format()
+      return 9
+    }
+
+    const weekStart = moment('2015-12-20')
+    const week = shallow(
+      <Week day={weekStart} showWeekNumber formatWeekNumber={weekNumberFormatter} />
+    )
+    const weekNumberElement = week.find(WeekNumber)
+
+    expect(firstDayReceived).to.equal(weekStart.format())
```

Comment:
very minor point but..
when i first read this it took me a while to work out why you were using .format(), here and on L97. I think you are doing it get a string so that value comparison works in the expectation.

I think a more intention revealing way to do this would be to use moments isSame() eg:
expect(recievedDay.isSame(weekStart)).to.be.true